app.js

```diff
@@ -135,19 +135,27 @@ const styles = StyleSheet.create({
   },
 })
 
+let settingsButtonActive = false
```

Comment:
Make this a state variable in App, and pass it into the LeftButtom function as a prop.